.github/workflows/continuous-integration-workflow.yml

```
@@ -0,0 +1,65 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
+
+jobs:
+  CI:
+    continue-on-error: ${{ matrix.distro == 'opensuse:latest' || matrix.distro == 'fedora:rawhide' || matrix.continue-on-error == true }}
```

```suggestion
    continue-on-error: true
```
I think it makes sense to always run all the tests.